test.md

```diff
@@ -0,0 +1,6 @@
+
+On the right ther is an elevator.
```

Comment:
**[misspell]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
"ther" is a misspelling of "there"